kofta/src/vscode-webview/components/BottomVoiceControl.tsx

```diff
@@ -40,7 +40,11 @@ export const BottomVoiceControl: React.FC<BottomVoiceControlProps> = ({
   const { muted, set } = useMuteStore();
   const [{ canSpeak, isCreator }] = useAtom(myCurrentRoomInfoAtom);
   const [settingsOpen, setSettingsOpen] = useState(false);
-  const [toggleOpen, newUnreadMessages, iAmMentioned] = useRoomChatStore(s => [
+  const [
+    toggleOpen,
+    newUnreadMessages,
+    iAmMentioned,
+  ] = useRoomChatStore((s) => [
```

Comment:
also before we go to far with chat logic, can you split mention state into its own store

Reply:
Sure